ui/components/multichain/pages/review-permissions-page/site-cell/site-cell.tsx

```diff
@@ -59,38 +63,42 @@ export const SiteCell: React.FC<SiteCellProps> = ({
   const [showEditAccountsModal, setShowEditAccountsModal] = useState(false);
   const [showEditNetworksModal, setShowEditNetworksModal] = useState(false);
 
-  const selectedAccounts = accounts.filter(({ address }) =>
+  const selectedAccounts = accounts.filter(({ caipAccountId }) =>
     selectedAccountAddresses.some((selectedAccountAddress) =>
-      isEqualCaseInsensitive(selectedAccountAddress, address),
+      isEqualCaseInsensitive(selectedAccountAddress, caipAccountId),
     ),
   );
-  const selectedNetworks = allNetworks.filter(({ chainId }) =>
-    selectedChainIds.includes(chainId),
+  const selectedNetworks = allNetworks.filter(({ caipChainId }) =>
+    selectedChainIds.includes(caipChainId),
   );
 
+  console.log({ allNetworks, selectedNetworks, selectedChainIds });
+
   const selectedChainIdsLength = selectedChainIds.length;
 
   // Determine the messages for connected and not connected states
   const accountMessageConnectedState =
     selectedAccounts.length === 1
       ? t('connectedWithAccountName', [
-          selectedAccounts[0].metadata.name || selectedAccounts[0].label,
+          selectedAccounts[0].internalAccount.metadata.name ||
+            selectedAccounts[0].internalAccount.label,
         ])
       : t('connectedWithAccount', [selectedAccounts.length]);
   const accountMessageNotConnectedState =
     selectedAccounts.length === 1
       ? t('requestingForAccount', [
-          selectedAccounts[0].metadata.name || selectedAccounts[0].label,
+          selectedAccounts[0].internalAccount.metadata.name ||
+            selectedAccounts[0].internalAccount.label,
         ])
       : t('requestingFor');
 
   const networkMessageConnectedState =
     selectedChainIdsLength === 1
-      ? t('connectedWithNetworkName', [selectedNetworks[0].name])
+      ? t('connectedWithNetworkName', [selectedNetworks[0]?.name])
       : t('connectedWithNetwork', [selectedChainIdsLength]);
   const networkMessageNotConnectedState =
     selectedChainIdsLength === 1
-      ? t('requestingForNetwork', [selectedNetworks[0].name])
+      ? t('requestingForNetwork', [selectedNetworks[0]?.name])
```

Comment:
why are these nullish coalesce needed now?

Reply:
seems like they are not. removing